Archipelago.MultiClient.Net.Analyzers/Archipelago.MultiClient.Net.Analyzers/Fixes/ItemFlagsInSwitchFixes.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Text;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Archipelago.MultiClient.Net.Analyzers.Fixes
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public class ItemFlagsInSwitchFixes : CodeFixProvider
+    {
+        public const string FixKeyConvertItemFlagsSwitch = "ConvertItemFlagsSwitch";
+
+        public override ImmutableArray<string> FixableDiagnosticIds => [
+            Constants.DiagnosticPrefix + "003"
+        ];
+
+        public override FixAllProvider? GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
+
+        public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            SyntaxNode? root = await context.Document.GetSyntaxRootAsync(context.CancellationToken)
+                .ConfigureAwait(false);
+            if (root == null)
+            {
+                return;
+            }
+
+            Diagnostic diagnostic = context.Diagnostics.First();
+            TextSpan span = diagnostic.Location.SourceSpan;
+            SwitchStatementSyntax? switchStatement = root.FindToken(span.Start).Parent?
+                .FirstAncestorOrSelf<SwitchStatementSyntax>();
+            if (switchStatement == null)
+            {
+                return;
+            }
+
+            List<SyntaxNode> caseTargets = new();
+            foreach (SwitchSectionSyntax switchSection in switchStatement.Sections)
+            {
+                IEnumerable<SyntaxNode> nodesToMove = switchSection.Labels
+                    .OfType<CaseSwitchLabelSyntax>()
+                    .SelectMany(label => label.DescendantNodes());
+
+                caseTargets.AddRange(nodesToMove);
+            }
+
+            context.RegisterCodeFix(
+                    CodeAction.Create(
+                        title: "Convert ItemFlags switch to if/else",
```

Comment:
```suggestion
                        title: "Convert to if/else",
```

This seems more stylistically correct